packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -230,12 +251,15 @@ export default class TableResize implements EditorPlugin {
         if (this.insertingState == ResizeState.None) {
             return;
         }
+
+        this.isRTL = getComputedStyle(this.editor.getDocument().body, 'direction') == 'rtl';
```

Comment:
this.isRTL is ued before inserter is created. Need a better place to do this.

Reply:
I moved it after "const inserter = fromHtml..."